src/components/services-view/services-view.controller.ts

```diff
@@ -192,7 +192,7 @@ export class ServicesViewController implements angular.IController {
         this.ctrl.filterConfig.resultsCount = this.ctrl.filteredItems.length;
         $('.toolbar-pf-results h5').text(this.ctrl.filterConfig.resultsCount + ' of ' + this.ctrl.totalCount + ' items');
       } else {
-        $('.toolbar-pf-results h5').text(this.ctrl.totalCount + ' items');
+        $('.toolbar-pf-results h5').text(this.ctrl.totalCount + ' item' + (this.ctrl.totalCount > 1 ? 's' : ''));
```

Comment:
Hi @spadgett, for 0 results it will read "0 of 15 items".
Also, sorry, snuck into this PR a fix for another small issue.